### ISSUE TITLE
eslint complaining about not finding libStorage

### ISSUE THREAD

**Issue by katlim-br** [m-radzikowski/aws-sdk-client-mock#102]
**Checklist**

- [x ] I have read [Caveats](https://github.com/m-radzikowski/aws-sdk-client-mock#caveats) documentation and didn't find a solution for this problem there.

**Bug description**

With v1.0.0

If I do this
```
in package.json
    "devDependencies": {
        "aws-sdk-client-mock": "^1.0.0"
    }

in mytest.spec.js
    const { mockClient } = require("aws-sdk-client-mock");
    const { mockLibStorageUpload } = require("aws-sdk-client-mock/libStorage");
```
eslint will complain about not finding the reference for libStorage, although when I run the tests, they run ok.

So the solution for now is to add a disable for next line, but it is still weird it shows up.
// eslint-disable-next-line import/no-unresolved

**Environment**

- Node version: 14
- Testing lib and version: jest 28
- Typescript version: js
- AWS SDK v3 Client mock version: 1.0.0
- AWS JS SDK libs and versions: 3.112
  -

**Comment by m-radzikowski** [comment]
Hey. I plan to remove the `mockLibStorageUpload` altogether in the next release. It caused far too many problems. See #105.

Please see the updated README on the `libStorage` mocking: https://github.com/m-radzikowski/aws-sdk-client-mock/blob/26311f15e8/README.md#lib-storage-upload

In short, all you have to do to mock the `libStorage` is to mock those two Commands:

```ts
const s3Mock = mockClient(S3Client);
s3Mock.on(CreateMultipartUploadCommand).resolves({UploadId: '1'});
s3Mock.on(UploadPartCommand).resolves({ETag: '1'});
```

**Comment by m-radzikowski** [comment]
Note: I've released the [2.0.0-beta.0](https://www.npmjs.com/package/aws-sdk-client-mock/v/2.0.0-beta.0) version without the `mockLibStorageUpload`. Please try it, manually mocking `CreateMultipartUploadCommand` and `UploadPartCommand`.

**Comment by katlim-br** [comment]
thanks for the answer
